content/blog/2018/03/2018-03-14-jep-200-lts.adoc

```diff
@@ -0,0 +1,155 @@
+---
+layout: post
+title: "Breaking change: Jenkins LTS 2.107.1 switches XStream / Remoting blacklists to whitelists (JEP-200)"
+tags:
+- core
+- security
+- remoting
+- upgrade
+author: oleg_nenashev
+---
+
+[NOTE]
+====
+This is a post about a major change in Jenkins, which is available starting
+from Jenkins 2.102 and Jenkins LTS 2.107.1.
+This is a change with a serious risk of regressions in plugins.
+If you are a Jenkins administrator, please read this blogpost and
+link:/doc/upgrade-guide/2.107/#upgrading-to-jenkins-lts-2-107-1[upgrade guidelines]
+*BEFORE* upgrading.
+====
+
+I would like to provide some heads-up about
+link:https://github.com/jenkinsci/jep/blob/master/jep/200/README.adoc[JEP-200],
+which is going to land in the new Jenkins LTS baseline.
+JEP-200 is “just the biggest incompatible change to Jenkins in recent memory" according to link:https://github.com/jglick[Jesse Glick],
+and I highly recommend to read guidelines below to minimize potential impact on your instances.
```

Comment:
BTW I am not sure the term _instance_ is widely recognized among Jenkins users in the field. It is a term we use internally to identify one `$JENKINS_HOME` but probably _installation_ or _service_ or something would be more familiar.